pages/getting-access.md

```diff
@@ -1,23 +1,71 @@
 
-!!! Warning  
-    Access to most CCR resources is restricted to UB and Roswell Park networks (either on campus or connected to their VPN services).  Users without access to UB or RPCI accounts should [contact CCR Help](help.md) for VPN access.
+## VPN Access
+
+Access to all CCR resources, including creating accounts is restricted to UB
+and Roswell Park networks. You must be either on campus or connected to
+[UBVPN](https://www.buffalo.edu/ubit/service-guides/connecting/vpn.html) or
+Roswell Park VPN. For more information on connecting to UBVPN [see here](https://www.buffalo.edu/ubit/service-guides/connecting/vpn/computer.html).
+If you do not have a UBIT account or a Roswell Park user account please
+[contact CCR Help](help.md) to inquire about VPN access.
 
 ## Creating a CCR user account
 
-Users with UB or Roswell Park email addresses can create themselves a CCR user account following [these directions](portals/idm.md#create-a-ccr-system-account).  Once your account is created, make sure to [follow the instructions for enabling two factor authentication](portals/idm.md#managing-two-factor-authentication).  Now it's time to request access to the resources or services you want to use.
+CCR resources are freely available to the UB and Roswell Park research
+communities. You must create an individual CCR account and enable two factor
+authentication if you wish to access any CCR resources. Please note that CCR
+user accounts are separate from your UBIT or Roswell Park accounts. 
+
+A CCR user account can be created quickly and easily by [by filling out the
+form here](https://idm.ccr.buffalo.edu/signup) using your UB or Roswell Park
+email address. Once your account is created you will need to verify your email
+address and enable two factor authentication. 
+
+## Enable Two Factor Authentication
+
+CCR requires all user accounts to have two factor authentication enabled. We
+support OTP, which stands for One-Time Passcodes and is a common form of two
+factor authentication (2FA). OTP is also known as app based authentication,
+software tokens, or soft tokens where an app on your phone will generate unique
+numeric passcodes as a second factor to provide increased account security.
+
+To enable two factor authentication on your account, follow these easy steps:
+
+1. Install an authenticator app on your phone. Authentication apps like Authy,
+   Google Authenticator, and Duo are supported. We recommend Duo as it's also used by UBIT.
+
+2. Login to CCR IDM portal [here](https://idm.ccr.buffalo.edu/) and click on
+   [OTP Tokens](https://idm.ccr.buffalo.edu/otp) in the side menu.
+
+3. Click on the "New Token" button and follow the directions to scan the QR
+   code using your authenticator app. Enter the six digit passcode generated by the app.
+
+You can also watch the video below which details the entire process:
+
+![type:video](https://youtube.com/embed/BDd-J1DAwsw)  
+
```

Comment:
I don't love that we now have this info 3 times between this, the 2fa page and the IDM portal page.  I don't like sending users on multiple clicks to different areas of the docs but I was hoping for more of a 1,2,3 steps to get access rather than all the instructions repeated on multiple pages.  I guess we'll have to agree to disagree on this.  :)

Reply:
Took another pass at cleaning this up. I removed the video from the getting access page and added a link to the 2fa page for more info. I also took a pass at cleaning up the ColdFront page so that it flows nicely from the getting access page. Let me know what you think!